mos_tests/neutron/python_tests/test_ovs_restart.py

```diff
@@ -895,3 +902,148 @@ def test_ovs_new_flows_added_after_restart(self):
+                                   self.server2_ip, timeout=None)
+
+        # make a list of all ovs agent ids
+        self.ovs_agent_ids = [
+            agt['id'] for agt in
+            self.os_conn.neutron.list_agents(
+                binary='neutron-openvswitch-agent')['agents']]
+        # make a list of ovs agents that resides only on controllers
+        controllers = [node.data['fqdn']
+                       for node in self.env.get_nodes_by_role('controller')]
+        ovs_agts = self.os_conn.neutron.list_agents(
+            binary='neutron-openvswitch-agent')['agents']
+        self.ovs_conroller_agents = [agt['id'] for agt in ovs_agts
+                                     if agt['host'] in controllers]
+
+    def test_ovs_restart_pcs_disable_enable_ping_private_vms(self):
+        """ Restart openvswitch-agents with pcs disable/enable on controllers.
+        [VLAN only] Check connectivity between private networks on different
+        routers
+        Testrail: 542666
+        Jira:     QA-375
+
+        Steps:
+        1. Update default security group if needed
+        2. Create CONFIG 1:
+        Network: test_net_05
+        SubNetw: test_net_05__subnet, 192.168.5.0/24
+        Router:  test_router_05
+        3. Create CONFIG 2:
+        Network: test_net_06
+        SubNetw: test_net_06__subnet, 192.168.6.0/24
+        Router:  test_router_06
+        4. Launch 'test_vm_05' inside 'config 1'
+        5. Launch 'test_vm_06' inside 'config 2'
+        6. Go to 'test_vm_05' console and send pings to 'test_vm_05'.
+        Pings should NOT go between VMs.
+        7. Disable ovs-agents on all controllers, restart service
+        neutron-plugin-openvswitch-agent on all computes, and enable
+        them back. To do this, launch the script against master node.
+        8. Wait 30 seconds, send pings from 'test_vm_05' to 'test_vm_06'
+        and check that they are still NOT successful.
+
+        Duration 5m
+
+        """
+        # Check that all ovs agents are alive
+        self.os_conn.wait_agents_alive(self.ovs_agent_ids)
+
+        # Disable ovs agent on all controllers
+        self.disable_ovs_agents_on_controller()
+
+        # Then check that all ovs went down
+        self.os_conn.wait_agents_down(self.ovs_conroller_agents)
+
+        # Restart ovs agent service on all computes
+        self.restart_ovs_agents_on_computes()
+
+        # Enable ovs agent on all controllers
+        self.enable_ovs_agents_on_controllers()
+
+        # Then check that all ovs agents are alive
+        self.os_conn.wait_agents_alive(self.ovs_agent_ids)
+
+        # sleep is used to check that system will be stable for some time
+        # after restarting service
+        time.sleep(30)
+
+        self.check_no_ping_from_vm(self.server1, self.instance_keypair,
+                                   self.server2_ip, timeout=None)
+
+        # check all agents are alive
+        assert all([agt['alive'] for agt in
```

Comment:
That step is not necessary for steps (from docstring). May be you need to add 9th step?